dcicutils/es_utils.py

```diff
@@ -86,3 +86,32 @@ def execute_lucene_query_on_es(client, index, query):
     except KeyError:
         logger.error('Searching index %s with query %s gave no results' % (index, query))
         return None
+
+
+def get_bulk_uuids_embedded(client, index, uuids):
```

Comment:
I'll have to think more on this, but I'll just ask the question aloud anyway: Are there security issues in accessing in bulk, since the individual items will have differing security characteristics?  Is that already handled in `client.mget` and not relevant here?

Also, if a problem results in getting one of these but not another, do you get a partial result? no result? a list of errors? ... Maybe again that is really an issue of mget, which I'll have to look into. I'm just thinking aloud here.

Reply:
Since this API is exposed through dcicutils and requires an ES client, only authenticated users can do this operation. It isn't mean to be an externally facing (public) API. If we want to provide this ability application side, there would need to be greater protections in place.